src/schoology-extractor/tests/test_lib/__init__.py

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
About "test_lib" directory name, not this file.  I haven't seen this pattern of prefixing module name being tested with "test_".  I've only seen directory name same as module test name.

Reply:
Fixed